beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -51,13 +54,20 @@ def from_config(cls, qconfig: QuantizerConfig, narrow_range: bool, half_range: b
 
 class Quantizer(NNCFOperation):
     """
-    Base class for all NNCF quantization operations
+    Base class for all NNCF quantization operations.
     """
-    def __init__(self):
+    def __init__(self, name: str, qspec: TFQuantizerSpec):
         """
-        Initializes internal NNCF quantization operation state
+        Initializes internal NNCF quantization operation state.
+
+        :param name: Unique operation name in algorithm scope.
+        :param qspec: Parameter struct required to initialize Quantizer.
         """
-        super().__init__()
+        super().__init__(name=name)
+        self.num_bits = qspec.num_bits
+        self.per_channel = qspec.per_channel
+        self.narrow_range = qspec.narrow_range
+        self._half_range = qspec.half_range
```

Comment:
Please, explain why this refactoring is needed.

Reply:
`SymmetricQuantizer` and `AsymmetricQuantizer` both have similar initialization procedure except  `self.signedness_to_force` attribute. I believe moving base quantizers attributes initialization will increase readability and decrease mistakes prob.

Reply:
I would disagree with your point of view.

Reply:
I reverted my changes, please check